manifests/e/emoacht/Monitorian/4.2.2/emoacht.Monitorian.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: emoacht.Monitorian
+PackageVersion: 4.2.2
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86

```

```suggestion
  Architecture: x86
  Scope: machine
```
The installation is machine-wide